bindings/Sofa/src/SofaPython3/Sofa/Core/CMakeLists.txt

```diff
@@ -67,15 +67,15 @@ if (NOT TARGET SofaPython3::Plugin)
 endif()
 
 find_package(SofaFramework REQUIRED)
-find_package(SofaBase REQUIRED)
-find_package(SofaCommon REQUIRED)
 find_package(SofaSimulation REQUIRED)
+find_package(SofaBaseVisual REQUIRED)
```

Comment:
I understand the need of BaseUtils (for requiredplugin I presume!) but why SofaBaseVisual ?

Reply:
@fredroy for this:

https://github.com/sofa-framework/SofaPython3/blob/f99e60dbd82df619d99a445f227efe257f32590a/bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_BaseCamera.cpp#L32

Reply:
A bit weird that something from a non-core module is bind into the Core module.
I guess this should be moved somewhere in the future for more consistency (but not the point of this PR)

Reply:
Absolutely